lib/transforms/autoprefixer.js

```diff
@@ -17,13 +17,17 @@ module.exports = (
       try {
         autoprefixerVersionStr = require('autoprefixer/package.json').version;
       } catch (e) {
-        e.message =
-          `autoprefixer transform: Found ${
-            cssAssets.length
-          } css asset(s) while --browsers option is active, ` +
-          `but no autoprefixer module is available. Please use npm to install autoprefixer in your project so ` +
-          `the autoprefixer transform can require it.\n${e.message}`;
-        throw e;
+        if (errorIfNotFound) {
+          e.message =
+            `autoprefixer transform: Found ${
+              cssAssets.length
+            } css asset(s) while --browsers option is active, ` +
+            `but no autoprefixer module is available. Please use npm to install autoprefixer in your project so ` +
+            `the autoprefixer transform can require it.\n${e.message}`;
+          throw e;
```

Comment:
Isn't the convention in other transforms where we do optional work that we emit a warning instead?